drivers/display/display_ili9806e_dsi.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_DISPLAY_ILITEK_ILI9806E_H_
+#define ZEPHYR_DRIVERS_DISPLAY_ILITEK_ILI9806E_H_
+
+#define GLCD_TIMINGS_NODE DT_CHILD(DT_NODELABEL(zephyr_lcdif), display_timings)
+
+#define GLCD_HTIMMING_SYNC       DT_PROP(GLCD_TIMINGS_NODE, hsync_len)
```

Comment:
Why name these GLCD_TIMINGS? They are not specific to the GLCD.

Reply:
I fixed it